### PR TITLE
Documentation tweaks

### PR DESCRIPTION
**Pull request by davelab6**
I updated the documentation a little more, merging contents from other files into the README, and putting pertinent commit lines into the changes file for the last 2 releases.

### CODE REVIEW

**Comment by moyogo** [comment]
Would it make sens to move the https://github.com/behdad/fonttools/wiki/FAQ to a file?

File: Doc/changes.txt [hunk 1]
```diff
@@ -1,4 +1,104 @@
-TTX/FontTools Version 2.4
+## TTX/FontTools Version 3.0
+
+2015-09-01 <https://github.com/behdad/fonttools/commit/58f86f318af8b2f8b553e8cceef678d6ce0017c5>
+
+- [Snippets] Add script for cmap subtable format conversion
+- [Snippets] Add `layout-features.py` that prints GSUB/GPOS features
+- [feaLib] Add lexer for OpenType feature files
+- [ttx] add support for dumping WOFF2 fonts to TTX
```
Comment (moyogo) [inline Doc/changes.txt]: This should be in the changelog as well "Add support for saving WOFF2 file format"

Comment (davelab6) [inline Doc/changes.txt]: Yep, I hate github wikis, much prefer markdown files in the github repo
(and moving from master to gh-pages, so they can be accessed online nicely
:)

**Comment by behdad** [comment]
There's a merge commit in there.  Please rebase.  We should curate a hand-written changes file.  The headline-of-commits isn't useful.

**Pull request closed without merging**